nrf_802154/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,16 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+Main branch
+***********
+
+Bug fixes
+=========
+
+* For delayed transmission on nRF5340 when passing NULL metadata the channel now defaults to the value in PIB.
+  It used to be 11, which was inconsistent with the behaviour on nRF52. (KRKNWK-13539)
+* Fixed incorrect CSL phase calculation (the calculated phase was too small). (KRKNWK-13782)
```

Comment:
```suggestion
* Fixed an issue with the incorrect CSL phase calculation caused by the calculated phase being too small. (KRKNWK-13782)
```

Reply:
This is not entirely what I meant, the calculated phase being too small is the error itself, not the root cause.

Would:
 "Fix an issue causing the calculated CSL phase to be too small" be better?

Reply:
For now I changed to "Fixed an issue causing the calculated CSL phase to be too small"

Reply:
Looks ok!